lib/screens/change_email.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:flutter/material.dart';
+import 'package:scoped_model/scoped_model.dart';
+import 'package:ofypets_mobile_app/scoped-models/main.dart';
+import 'package:ofypets_mobile_app/utils/constants.dart';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import 'package:shared_preferences/shared_preferences.dart';
+
+class EmailEdit extends StatefulWidget {
+  @override
+  _EmailEditState createState() => _EmailEditState();
+}
+
+class _EmailEditState extends State<EmailEdit> {
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  String _email = 'sagar@gmail.com';
+  final TextEditingController _textFieldController = TextEditingController();
+  bool _fetching_email = true;
```

Comment:
Please use lowerCamelCase here as well. For local variables you can use something like _fetchingEmail, etc.